ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1017,6 +1017,30 @@ def cdread(self, *args, **kwargs):
 
         self.input(fname, **kwargs)
 
+    @wraps(_MapdlCore.tbft)
+    def tbft(self, oper='', id_='', option1='', option2='', option3='', option4='', option5='', option6='', option7='', **kwargs):
+
+        if oper.lower() == 'eadd':
+            # Option 2 is a file and option 4 is the directory.
+            # Option 3 is be extension
+            option3 = option3.replace('.', '')
+            filename = os.path.join(option4, option2 + '.' + option3)
+
+            if self._local:
+                if not os.path.exists(file) and filename not in self.list_files:
+                    raise Exception(f"File '{filename}' could not be found.")
```

Comment:
Always use the right exception type. See [Python Exceptions](https://docs.python.org/3/library/exceptions.html)
```suggestion
                    raise FileNotFoundError(f"File '{filename}' could not be found.")
```